### PR TITLE
Update package metadata for Amazon.Lambda.Annotations

### PR DESCRIPTION
**Pull request by normj**
*Description of changes:*
Add a README and icon to Amazon.Lambda.Annotations package.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Libraries/src/Amazon.Lambda.Annotations/README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Amazon.Lambda.Annotations
+
+The Lambda Annotation is a programming model for writing .NET Lambda function. At a high level the programming model allows for
```
Comment (ganeshnj) [inline Libraries/src/Amazon.Lambda.Annotations/README.md:3]:
```suggestion
The Lambda Annotations is a programming model for writing .NET Lambda function. At a high level the programming model allows
```

File: Libraries/src/Amazon.Lambda.Annotations/README.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
+# Amazon.Lambda.Annotations
+
+The Lambda Annotation is a programming model for writing .NET Lambda function. At a high level the programming model allows for 
+idiomatic .NET coding patterns and uses C# source generator technology to bridge the gap between the Lambda programming model
```
Comment (ganeshnj) [inline Libraries/src/Amazon.Lambda.Annotations/README.md:4]:
```suggestion
idiomatic .NET coding patterns and uses [C# Source Generators](https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/source-generators-overview) to bridge the gap between the Lambda programming model 
```

File: Libraries/src/Amazon.Lambda.Annotations/README.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    * Configures the Lambda function to be called from an API Gateway REST API. The HTTP method and resource path are required to be set on the attribute.
+* HttpApi
+    * Configures the Lambda function to be called from an API Gateway HTTP API. The HTTP method, HTTP API payload version and resource path are required to be set on the attribute.
+
+### Parameter Attributes
+
+* FromHeader
+    * Map method parameter to HTTP header value
+* FromQuery
+    * Map method parameter to query string pareamter
```
Comment (ganeshnj) [inline Libraries/src/Amazon.Lambda.Annotations/README.md:143]:
```suggestion
    * Map method parameter to query string parameter
```

File: Libraries/src/Amazon.Lambda.Annotations/README.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+* FromRoute
+    * Map method parameter to resource path segement
```
Comment (ganeshnj) [inline Libraries/src/Amazon.Lambda.Annotations/README.md:145]:
```suggestion
    * Map method parameter to resource path segment
```

File: Libraries/src/Amazon.Lambda.Annotations/README.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+List of .NET attributes currently supported.
+
+
+* LambdaFunction
+    * Placed on a method. Indicates this method should be exposed as a Lambda function.
+* LambdaStartup
+    * Placed on a class. Indicates this type should be used as the startup class and is used to configure the dependency injection and middleware. There can only be one class in a Lambda project with this attribute.
+
+### Event Attributes
```
Comment (ganeshnj) [inline Libraries/src/Amazon.Lambda.Annotations/README.md:132]: Should include one liner for no event attached lambda function. They can also be written using Annotations and take the advantage of template generation, DI etc.

**Review by ganeshnj — CHANGES_REQUESTED** [review]

File: Libraries/src/Amazon.Lambda.Annotations/README.md (hunk 5 above)
Comment (normj) [reply Libraries/src/Amazon.Lambda.Annotations/README.md:132]: Add a description to the event attribute section and mentioned that you don't have to have an event attribute.

**Review by ganeshnj — COMMENTED** [review]
Changes look good.

dotnet pack is failing now

```
Error: /home/runner/.dotnet/sdk/5.0.403/Sdks/NuGet.Build.Tasks.Pack/build/NuGet.Build.Tasks.Pack.targets(221,5): error NU5039: The readme file 'docs\readme.md' does not exist in the package. [/home/runner/work/aws-lambda-dotnet/aws-lambda-dotnet/Libraries/src/Amazon.Lambda.Annotations/Amazon.Lambda.Annotations.csproj]
Error: Process completed with exit code 1.
```

File: Libraries/src/Amazon.Lambda.Annotations.nuspec [hunk 6]
```diff
@@ -11,9 +11,14 @@
         <dependencies>
             <group targetFramework=".NETStandard2.0"/>
         </dependencies>
+		<icon>images\icon.png</icon>
+		<readme>docs\README.md</readme>
     </metadata>
 
     <files>
+		<file src="..\..\icon.png" target="images\" />
+		<file src="Amazon.Lambda.Annotations\README.md" target="docs\" />
```
Comment (ganeshnj) [inline Libraries/src/Amazon.Lambda.Annotations.nuspec:20]: case sensitivity ;)

**Review by ganeshnj — APPROVED** [review]

**Review by 96malhar — APPROVED** [review]

**Pull request merged**